seeders/20220216081856-demo-user.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    await queryInterface.bulkInsert(
+      'user',
+      [
+        {
+          id: 1,
+          firstName: 'Fabien',
+          lastName: 'NIRINGIYIMANA',
```

Comment:
Instead of using your own credentials, plz use a generic user as the default user, Ex: `admin`, `admin@admin.com`.

Reply:
Done!